client/src/components/CardDialog/CheckList/CheckList.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import Input from '@material-ui/core/Input';
+import InputLabel from '@material-ui/core/InputLabel';
+import MenuItem from '@material-ui/core/MenuItem';
+import FormControl from '@material-ui/core/FormControl';
+import ListItemText from '@material-ui/core/ListItemText';
+import Select from '@material-ui/core/Select';
+import Checkbox from '@material-ui/core/Checkbox';
+import checkListStyles from './checkListStyles';
+
+const ITEM_HEIGHT = 48;
+const ITEM_PADDING_TOP = 8;
+const MenuProps = {
+  PaperProps: {
+    style: {
+      maxHeight: ITEM_HEIGHT * 4.5 + ITEM_PADDING_TOP,
+      width: 250,
+    },
+  },
+};
+
+const names = ['Buy supplies', 'Contact teammates', 'Research', 'Book study hours'];
+
+export default function MultipleSelect() {
+  const classes = checkListStyles();
+  const [personName, setPersonName] = React.useState<string[]>([]);
+
+  const handleChange = (event: React.ChangeEvent<{ value: unknown }>) => {
+    setPersonName(event.target.value as string[]);
+  };
```

Comment:
Slightly confusing here; `personName` sounds like a single name but it's a string array. Could you rename it for better legibility?